main.py

```diff
@@ -1,10 +1,36 @@
 import init_django_orm  # noqa: F401
-
+from json import load
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = load(file)
+    for key, value in data.items():
+        race, created = Race.objects.get_or_create(
```

Comment:
Since `created` is not used, omit it here and in the other places
```suggestion
        race, _ = Race.objects.get_or_create(
```